src/annotations/AnnotationParser.php

```diff
@@ -167,6 +178,57 @@ public function parse($source, $path)
                     if ($type == T_FUNCTION) {
                         $state = self::METHOD_NAME;
                     }
+                    if ($type == T_USE) {
+                        $state = self::CLASS_USE_CLAUSE;
+                        $use = '';
+                    }
+                    break;
+
+                case self::CLASS_USE_CLAUSE:
+                    if ($type === self::CHAR) {
+                        if ($str === ';') {
+                            $state = self::SCAN_CLASS;
+                        } elseif ($str === '{') {
```

Comment:
Probably doesn't matter in this case (and neither does the matching `$nesting--`) because code can't occur inside these braces - they're just part of the use-clause syntax.